src/fsharp/symbols/SymbolHelpers.fs

```diff
@@ -384,11 +384,12 @@ module internal SymbolHelpers =
         | Item.SetterArg (_,item) -> rangeOfItem g preferFlag item
         | Item.ArgName (id,_, _) -> Some id.idRange
         | Item.CustomOperation (_,_,implOpt) -> implOpt |> Option.bind (rangeOfMethInfo g preferFlag)
+        | Item.ImplicitOp (_, {contents = Some(TraitConstraintSln.FSMethSln(_, vref, _))}) -> Some vref.Range
         | Item.ImplicitOp _ -> None
-        | Item.NewDef id -> Some id.idRange
         | Item.UnqualifiedType tcrefs -> tcrefs |> List.tryPick (rangeOfEntityRef preferFlag >> Some)
         | Item.DelegateCtor typ 
         | Item.FakeInterfaceCtor typ -> typ |> tryNiceEntityRefOfTy |> Option.map (rangeOfEntityRef preferFlag)
+        | Item.NewDef _ -> None
```

Comment:
I think we should change this ``Item.NewDef`` case back to the old code unless we know the specific reason it's been changed. 

I don't know of a reason why the change would be harmful. The one thing I can think of is that the original author (was it @saul?) wanted "Go to definition" on an "override" (e.g. ``Dispsoe`` in ``override x.Dispose() = ...``) to go to the C# definition of the overridden method.  That would be sufficient to make the change I suppose.   But I'm not sure the change has the effect of implementing that